scripts/windows_build_setup.ps1

```diff
@@ -72,7 +72,7 @@ export PATH=`"/c/BuildTools/MSBuild/Current/Bin:/c/BuildTools/VC/Tools/MSVC/14.2
 # Stop the script after first error
 $ErrorActionPreference = 'Stop'
 # Version of Qt SDK available form aqt
-$QtVersion = "5.14.2"
+$QtVersion = "5.15.2"
```

Comment:
Was this tested? Does this actually work?

Reply:
The windows build was generated on this PR. But i'm not 100% sure if jenkins did use this branch script?